### ISSUE TITLE
Your XSD Schema is outdated... How to get IntelliSense in XML files?

### ISSUE THREAD

**Issue by pushqrdx** [NativeScript/NativeScript#6327]
The Schema at `http://schemas.nativescript.org/tns.xsd` seems to be outdated as it doesn't support this code from the docs

```
    <ActionBar class="action-bar">
        <Label class="action-bar-title" text="Home">
            
        </Label>
    </ActionBar>
```

The tooling for NativeScript is pretty poor and it's petty because I really like the framework 😐

**Comment by NickIliev** [comment]
@pushqrdx thanks for bringing that into our attention - we will take a look at this one and I will post any related info here.

Meanwhile, you could use a VSCode extension to get not only tag insertion but the insertion of tag along with their specific attributes. For example, here is [an XML one](https://marketplace.visualstudio.com/items?itemName=tsvetan-ganev.nativescript-xml-snippets) and here is one for [Angular based applications](https://marketplace.visualstudio.com/items?itemName=wwwalkerrun.nativescript-ng2-snippets).

**Comment by pushqrdx** [comment]
@NickIliev having the XSD will just improve the workflow drastically. l use vue and having no intellisense support at all makes for a very unpleasant experience. especially having to jump back and forth between the code and the docs to figure out each component props/events.

**Comment by ehigiepaul** [comment]
@pushqrdx that's true the experience could be worst if they dont fix that issue. I have created a extension for VScode to handle this problem with snippet with all the attributes you need and more nativescript feature am still working on. Try  [Nativescript Extend](https://marketplace.visualstudio.com/items?itemName=Paul-Ehigie-Paul.nativescript-extend) I promise you wont regret it.

**Comment by pushqrdx** [comment]
@ehigiepaul It's so ironic that the community provides better tooling than the team xD

**Comment by ehigiepaul** [comment]
I have updated the XSD scheme file but how to integrate it into vscode is a problem for me. And am planing to add it to my extension with more features. Please can anyone tell me how to add a scheme to vscode.

**Comment by pushqrdx** [comment]
@ehigiepaul VSCode should, out of the box, provide intellisense for XSD namespaces just by importing them at the top of any XML file.

**Comment by ehigiepaul** [comment]
Am done with the update xsd schema and it now added into [Nativescript Extend](https://marketplace.visualstudio.com/items?itemName=Paul-Ehigie-Paul.nativescript-extend)
try it out.
